src/SFA.DAS.AANHub.Database/Scripts/AddLeavingReasonData.sql

```diff
@@ -0,0 +1,44 @@
+﻿
+CREATE TABLE #TempLeavingReason
+(
+    [Id] INT NOT NULL,
+    [Category] NVARCHAR(100) NULL, 
+    [Description] NVARCHAR(200) NULL, 
+    [Ordering] INT NULL
+);
+
+
+INSERT INTO #TempLeavingReason VALUES 
+(1, 'What are your reasons for leaving the network', 'I am unable to commit the time required to deliver my role', 1),

```

Comment:
Can the categories be single word instead of full text please, that will allow us to group thing easily, the text itself can be hardcoded on the view. 
Suggested values: Reason, Benefit, Experience

Reply:
Instead of experience we can use Ratings